README.md

```diff
@@ -62,11 +55,14 @@ USAGE:
    ipget [global options] command [command options] [arguments...]
 
 VERSION:
-   0.2.0
+   0.5.0
 
 COMMANDS:
 GLOBAL OPTIONS:
    --output value, -o value  specify output location
+   --node temp, -n temp      specify ipfs node strategy ('local', 'spawn', temp or 'fallback') (default: "fallback")
+   --peers value, -p value   specify a set of IPFS peers to connect to
+   --progress                show a progress bar
```

Comment:
Hmmm... looks like I made a mistake not bumping the version number in main.go to v0.6.0. Let's make a separate PR with this README change and bumping main.go to v0.6.1 that one I merge I can tag.

Reply:
I think this is still correct to have in this PR since I changed it from `0.2.0` which was totally wrong, to `0.5.0` which matches the current version.  A subsequent PR after that should bump the version in both the readme and the code.

Reply:
0.5.0 is the wrong version too. I've already release v0.6.0, but forgot to bump the main.go file. Because this is a bit weird we should just put this in a new PR to do v0.6.1 and then we can make other changes like #80 and do followup releases.

Reply:
Done.  I can rebase it all into one commit if you like.

Reply:
SGTM